app/components/ui-table/cell/admin/events/event-no-front.js

```diff
@@ -0,0 +1,5 @@
+import classic from 'ember-classic-decorator';
+import Component from '@ember/component';
+
+@classic
+export default class EventNoFront extends Component {}
```

Comment:
As we discussed previously, no new component will be made classic, classic has been deprecated

Reply:
```
import Component from '@glimmer/component';

export default class EventNoFront extends Component {}

```
![Screenshot from 2021-01-09 20-14-44](https://user-images.githubusercontent.com/56407566/104094517-572d2000-52b7-11eb-9b2e-1233baf00bf3.png)

Reply:
why?

Reply:
Write the component using new syntax. Remove the JS file, it is useless

Reply:
ok

Reply:
What is meaning by new syntax? I am using `<UiCheckbox @class="toggle" @checked={{this.extraRecords.isDemoted}} @onChange={{action this.props.actions.toggleDemoted this.record}} />` and it is giving error without js file

Reply:
Octane syntax